manifests/plugins/ep_mypads.pp

```diff
@@ -0,0 +1,30 @@
+class etherpad::plugins::ep_mypads {
+  etherpad::plugins::common { 'ep_mypads' :
```

Comment:
this is a private class, right? Please add [assert_private()](https://forge.puppet.com/puppetlabs/stdlib#assert_private)

Reply:
Yes, is a private class. I have adding `assert_type()`